src/main/webapp/i18n/de/exercise-actions.json

```
@@ -54,6 +54,10 @@
                 "button": "Im Git Client klonen",
                 "helpIcon": "Dies öffnet das Repository in Sourcetree oder Tower. Falls du diese nicht installiert hast, kannst du das Repository in deiner IDE oder dem Terminal klonen."
             },
+            "cloneIntellij": {
+                "button": "In IntelliJ klonen",
+                "helpIcon": "Dies öffnet das Repository in IntelliJ. Falls du IntelliJ nicht installiert hast, kannst du das Repository im Terminal klonen."
```

```suggestion
                "helpIcon": "Dies öffnet das Repository in IntelliJ. Falls du IntelliJ nicht installiert hast, kannst du das Repository anderweitig klonen."
```
Some students also use Eclipse or VsCode so the tooltip shouldn't sound like IntelliJ or the terminal are the only two options. (Same for the English version)

---

do we even need the helpIcon? 
Im just imaging the state where the VSCode clone button and the online IDE button is added as well and it will get crowded with all the helpicons

---

The first sentence alone would not add anything, so I agree with Dennis, that we can get rid of the help icon for this case. The help icon for the "git client" might still be helpful since new students will hear "git" for the first time and will not know if their IDE is also considered a git client